src/logic/basic.lean

```diff
@@ -1302,7 +1297,8 @@ propext (forall_prop_congr hq _)
 @[simp] lemma forall_true_left (p : true → Prop) : (∀ x, p x) ↔ p true.intro :=
 forall_prop_of_true _
 
-@[simp] lemma forall_false_left (p : false → Prop) : (∀ x, p x) ↔ true :=
+-- This is superceded by `is_empty.forall_iff`.
+private lemma forall_false_left (p : false → Prop) : (∀ x, p x) ↔ true :=
```

Comment:
It looks like you've already removed `forall_false_left` except for two uses. Can it be removed completely?

It might be nice to include in a docstring comment for `forall_true_left` that `is_empty.forall_iff` is the `false` version.

(A similar comment goes for `exists_false_left`.)

Reply:
I've removed these theorems entirely and added the comments you suggested.